qiskit_metal/qlibrary/qubits/JJ_Manhattan.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+#from math import *
+from math import sin, cos
+from qiskit_metal import draw, Dict
+from qiskit_metal.qlibrary.core.base import QComponent
+import numpy as np
+
+
+class JJ_Manhattan(QComponent):
+    """
+    The base "JJ_Manhattan" inherits the "QComponent" class.
+
+    This creates a "Manhattan"-style Josephson Junction consisting
```

Comment:
Awesome, maybe add a reference to a paper, such as the Dolan and Pop papers I sent over.

Reply:
Done!